python/setup.py

```diff
@@ -45,7 +45,7 @@
         "gpsd-py3==0.3.0",
         "paho-mqtt==1.6.1",
         "pyGeoTile==1.0.6",
-        "pytest==7.1.0",
+        "pytest==7.1.1",
```

Comment:
AS seen IRL, this should probably be offloaded to a separate setting so that we
can build a relase package without requiring testing facilities.

Reply:
I revert